docker/ol-docker-start.sh

```diff
@@ -28,8 +28,10 @@ su postgres -c "/etc/init.d/postgresql start"
 su openlibrary -c "scripts/infobase-server conf/infobase.yml 7000" &
 
 # wait unit postgres is ready, then reindex solr
+# also block until reindexing is completed since later services require psql
+# to be available anyways
 export -f reindex-solr
-su openlibrary -c "until pg_isready; do sleep 5; done && reindex-solr localhost $CONFIG" &
+su openlibrary -c "until pg_isready; do sleep 5; done && reindex-solr localhost $CONFIG"
```

Comment:
Is there are reason to block on this? The idea was to allow the re-indexing to take place in the background to allow the web site to spin up as fast as possible.

Reply:
I ran into problems with the infogami install task being unable to reach postgres, since it was still starting up and not accepting connections. This seemed like a reasonable solution given infogami requires postgres to be available to function afaict. I could try some sort of waiting/polling mechanism from within python, but I'm not sure where that would live to be honest. I say that because the startup/init process for infogami isn't super clear to me.

Reply:
If reindexing needs to take place in the background then we can take it out of that one liner and just make the loop blocking...

```shell
# ...
su openlibrary -c "until pg_isready; do sleep 5; done"

su openlibrary -c "reindex-solr localhost $CONFIG" &

# ...
```

This would allow reindexing to take place after postgres in is available, but in the background. Also allows the infogami install task/command to work since postgres will already be available.